modules/fscloud/main.tf

```diff
@@ -0,0 +1,61 @@
+##############################################################################
+# Get Cloud Account ID
+##############################################################################
+
+data "ibm_iam_account_settings" "iam_account_settings" {
+}
+
+##############################################################################
+# Create CBR Zone
+##############################################################################
+module "cbr_zone" {
```

Comment:
this module should not create the CBR zone

Reply:
Relocated CBR to example